### PR TITLE
Islamic star parts 1 & 2 consolidated.

### PR DESCRIPTION
**Pull request by kfahn22**
Here is my attempt at consolidation.  Let me know if this is ok or if changes are needed.

### CODE REVIEW

File: content/videos/challenges/54.2-star-patterns-update-law-of-sines/index.json [hunk 1]
```diff
@@ -1,28 +1,37 @@
...
-    "videoNumber": "54.1",
-    "videoId": "sJ6pMLp_IaI",
-    "date": "2017-02-01",
+    "description": "In this bonus super-sized coding challenge, I work through visualizing the Islamic Star Patterns in p5.js.",
+    "videoNumber": "54.2",
+    "videoId": "lobJ9gzbLo8",
+    "date": "2017-02-14",
     "languages": ["p5.js", "javascript"],
     "topics": [
         "JavaScript",
```
Comment (klinegareth) [inline content/videos/challenges/54.2-star-patterns-update-law-of-sines/index.json:9]: Maybe from here on out, anything listed in `languages` should be left out of `topics`

**Comment by kfahn22** [comment]
I see that there are conflicts with 54.1.  I am wondering whether it makes sense to delete that challenge 54.1  and then merge combined?

**Comment by klinegareth** [comment]
@kfahn22 I think that's a good idea!

**Comment by shiffman** [comment]
Yes, all good! @kfahn22 let us know if you need help or @klinegareth feel free to go ahead and resolve the conflicts! (Although maybe it's tricky, this is the thing I was mentioning that I couldn't figure easily figure out how to do if the branch is in a fork from a different account!)

**Comment by kfahn22** [comment]
It has occurred to me that maybe it would be easier to just leave 54.1 and edit the index.json file to reference part 2.

**Comment by kfahn22** [comment]
I am closing this one and creating two more.  One to delete 54.1 and a second to add the combined challenge.

**Pull request closed without merging**